DeviceSecurityPkg/Test/DeployCert/TestRootCer.c

```diff
@@ -1,10 +1,10 @@
-/** @file
-
-  Copyright (c) 2020, Intel Corporation. All rights reserved.<BR>
-  SPDX-License-Identifier: BSD-2-Clause-Patent
-
-**/
-GLOBAL_REMOVE_IF_UNREFERENCED UINT8 TestRootCer[]= {0x30, 0x82, 0x05, 0x19, 0x30, 0x82, 0x03, 0x01, 0xA0, 0x03, 0x02, 0x01, 0x02, 0x02, 0x14, 0x1D,
+/** @file
+
+  Copyright (c) 2020, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+GLOBAL_REMOVE_IF_UNREFERENCED UINT8 TestRootCer[]= {0x30, 0x82, 0x05, 0x19, 0x30, 0x82, 0x03, 0x01, 0xA0, 0x03, 0x02, 0x01, 0x02, 0x02, 0x14, 0x1D,
```

Comment:
please upload key and cert - https://github.com/jyao1/edk2/tree/DeviceSecurity/DeviceSecurityPkg/Test/DeployCert/Key